.rhiza/tests/stress/test_makefile_stress.py

```
@@ -0,0 +1,145 @@
+
+@pytest.mark.stress
+def test_concurrent_help_invocations(root: Path, concurrent_workers: int):
+    """Test concurrent invocations of make help.
+
+    Verifies that multiple simultaneous help invocations don't cause
+    race conditions or resource conflicts.
+    """
+
+    def run_help():
+        """Run make help and return success status."""
+        result = subprocess.run(  # nosec
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)